app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -75,6 +76,10 @@ public boolean onNavigationItemSelected(MenuItem item) {
             case R.id.item_charges:
                 replaceFragment(ClientChargeFragment.newInstance(clientId), false,  R.id.container);
                 break;
+            case R.id.item_transactions:
+                replaceFragment(SavingAccountsTransactionFragment.newInstance(21),
```

Comment:
@dilpreet96 You have to add this fragment in SavingsAccountDetails Toolbar menu item.